client/extensions/wp-super-cache/components/debug/index.jsx

```diff
@@ -44,13 +57,61 @@ class DebugTab extends Component {
 			handleSubmitForm,
 			isRequesting,
 			isSaving,
+			siteId,
 			translate,
 		} = this.props;
 
 		return (
 			<div>
+				<QueryDebugLogs siteId={ siteId } />
 				<form>
-					<SectionHeader label={ translate( 'Debug' ) }>
+					<SectionHeader label={ translate( 'Debug' ) } />
+					<Card>
+						<FormToggle
+							checked={ !! wp_super_cache_debug }
+							disabled={ isRequesting || isSaving }
+							onChange={ handleAutosavingToggle( 'wp_super_cache_debug' ) }>
+							{ translate( 'Enable Debugging' ) }
+						</FormToggle>
+					</Card>
+
+					<SectionHeader label={ translate( 'Debug Logs' ) } />
+					<Card>
+						<table>
+							<thead>
+								<tr>
+									<th>{ translate( 'Filename' ) }</th>
```

Comment:
Hi! I've found a possible matching string that has already been translated **98** times:
`translate( 'Empty filename' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wpsc-debug-log-buttons)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).